marlowe-playground-client/src/Language/Javascript/Monaco.purs

```diff
@@ -12,16 +13,16 @@ languageExtensionPoint = LanguageExtensionPoint { id: "typescript" }
 refLabel :: RefLabel
 refLabel = RefLabel "jsEditor"
 
-settings :: forall m. (Editor -> m Unit) -> Settings m
+settings :: forall m. MonadEffect m => (Editor -> m Unit) -> Settings m
```

Comment:
I don't see anything in this definition that requires `MonadEffect` - you could probably drop this constraint in this signature